pkg/apis/kops/cluster.go

```diff
@@ -349,6 +352,18 @@ type ClusterSubnetSpec struct {
 	Type SubnetType `json:"type,omitempty"`
 }
 
+type EgressProxySpec struct {
+	HTTPProxy     HTTPProxySpec `json:"httpProxy,omitempty"`
+	ProxyExcludes string        `json:"excludes,omitempty"`
+}
+
+type HTTPProxySpec struct {
```

Comment:
Supporting a seperate HTTPS is why I constructed the objects this way.  I took it out when I was cleaning up just because I wasn't using it, so it was unexecuted code.  Easy to put back in.